.agent/specs/312.md

```
@@ -0,0 +1,492 @@
+   │        ├─ ApproveButton
+   │        ├─ RejectButton
+   │        └─ ApproveIntentDialog [NEW] → AlertDialog
+   │             ├─ DialogTitle ("Intent 승인" / "Intent 반려")
+   │             ├─ IntentInfoPreview
+   │             ├─ ConfirmDescription
+   │             └─ DialogActions (확인 / 취소)
+   ├─ InfoCards (4-grid)
+   └─ JsonCards (sourceClusterRef, entryConditionJson, etc.)
+```
+
+#### 신규 컴포넌트
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)